src/coreclr/vm/codeman.h

```diff
@@ -69,6 +69,10 @@ Module Name:
 #include "gcinfo.h"
 #include "eexcp.h"
 
+#if defined(TARGET_ARM64)
+extern "C" uint64_t GetSveLengthFromOS();
```

Comment:
Could you please move this to the .cpp file (e.g. next to `GetDataCacheZeroIDReg`). It does not need to be in the header file.

Reply:
Done.